src/constants/Name.ts

```diff
@@ -142,10 +142,16 @@ export const getRandomName = () => {
   return `${randomAdjective} ${randomCreature}`;
 };
 
-export const getInitials = (name: string): string =>
-  name
-    .split(" ")
-    .map((n) => n[0])
-    .join("");
+export const getInitials = (name: string): string => {
+  // return first char of each word in name
+  if (name.indexOf(" ") > -1) {
```

Comment:
Should do the same. But I have not tested it.
```suggestion
  if (name.indexOf(" ") > -1) {
      var matches = str.match(/\b(\w)/g);
      var result = matches.join('');
  }
```

Reply:
Instead of `str.` you need to replace it with `name.` ;)

Reply:
nice one. Used it.